src/providers/xyz.js

```diff
@@ -0,0 +1,32 @@
+var d3 = require("d3");
+
+function XYZProvider(options) {
+  this.format = options.format;
+  this.tileCache = {};
+  this.urlTemplate = options.urlTemplate;
+}
+
+XYZProvider.prototype = {
+  getTile: function(tilePoint, callback){
+    var tileData = this.tileCache[tilePoint.zoom + ":" + tilePoint.x + ":" + tilePoint.y];
```

Comment:
Sorry, I did not explain myself well. I meant that setCartoCSS triggers an invalidation of the contents of the svg tile, so that geometries are redrawn using the new styles. The ideal method will be to update the style in a "d3 fashion", just modifying the models, but we're tracking that from #28.